docs/docs/ui-setup-own-components.md

```diff
@@ -96,9 +96,9 @@ import { useState } from 'react'
 import { SearchInput, Facets } from './components'
 
 const QUERY = gql`
-    query resultSet($query: String, $filters: [FiltersSet], $page: PageInput) {
+    query resultSet($query: String, $filters: [FiltersSet]) {
```

Comment:
hey one thing here is there are new variables for query which searchkit client passes via variables. See here https://github.com/searchkit/searchkit/blob/next/examples/next/components/index.jsx#L32,L47

Could you include them too?